src/gui/tray/Window.qml

```diff
@@ -322,6 +372,10 @@ Window {
                                 y: currentAccountStateIndicatorBackground.y + 1
                                 sourceSize.width: Style.accountAvatarStateIndicatorSize
                                 sourceSize.height: Style.accountAvatarStateIndicatorSize
+
+                                Accessible.role: Accessible.Indicator
+                                Accessible.name: UserModel.isUserConnected(UserModel.currentUserId()) ? qsTr("Connected") : qsTr("Disconnected")
+                                Accessible.description: qsTr("Icon that indicates connection state for the currently selected account")

```

Comment:
Here I’d also just say "Connected" and "Not connected" and no description needed.